permafrost/permissions.py

```diff
@@ -52,12 +52,12 @@ def has_all_permissions(request, check_list=[]):
     """
     Checks if request.user the given list of permission on the current request.site
     """
-    if not check_list:
```

Comment:
So the perms are now, if the user is super_user, always return true.  Just confirming the logic change.